src/tests/test_kube_api.py

```diff
@@ -107,19 +107,19 @@ def kube_api_test(
             base_domain=global_variables.base_dns_domain,
         )
         proxy = self.setup_proxy(nodes, cluster_config, proxy_server)
-
-        if is_disconnected:
-            log.info("getting ignition and install config override for disconnected install")
-            ca_bundle = self.get_ca_bundle_from_hub(spoke_namespace)
-            self.patch_install_config_with_ca_bundle(cluster_deployment, ca_bundle)
-            ignition_config_override = self.get_ignition_config_override(ca_bundle)
-        else:
-            ignition_config_override = None
-
+        ignition_config_override = None
         infra_env = InfraEnv(api_client, f"{cluster_name}-infra-env", spoke_namespace)
         infraenv = infra_env.create(
             cluster_deployment, secret, proxy, ignition_config_override, ssh_pub_key=cluster_config.ssh_public_key
         )
+        if is_disconnected and cluster_config.registry_ca_path is not None:
+            log.info("setting additional trust bundle for disconnected install")
+            registry_ca = None
```

Comment:
We lost the call to `ca_bundle = self.get_ca_bundle_from_hub(spoke_namespace)`

Reply:
This is no longer needed:
* `spoke_namespace` is generated dynamically, so user can't  create CA bundle beforehand
* `ca_bundle` is being injected as Ignition override by `infra_env.patch` too

Reply:
>spoke_namespace is generated dynamically, so user can't create CA bundle beforehand

Not sure I understand that. User might be a script / automation in assisted_test_infra that knows the dynamically generated spoke_namespace. Are you saying this code never worked?

>ca_bundle is being injected as Ignition override by infra_env.patch too

`ca_bundle` doesn't exist in your PR anymore

Reply:
>Are you saying this code never worked?

It worked -  if there was a way to create the necessary configmap via some other method, but there wasn't any.

>ca_bundle doesn't exist in your PR anymore

Patching infra env would automatically create an Ignition override, no other action required